### ISSUE TITLE
use another package than twitteR for the docs?

### ISSUE THREAD

**Issue by maelle** [nealrichardson/httptest#59]
I was thinking about this whilst looking at rtweet docs and seeing that the twitteR repo is archived.

**Comment by nealrichardson** [comment]
Yeah, you're right, it seems dated to use an inactive package. Do you have a suggestion of something better?

**Comment by maelle** [comment]
Nothing in particular but what'd be cool would be for your minimal demo to be a real PR (in the vignette you'd have the narrative, but then there'd really be httptest in the package).
Maybe that rOpenSci package you reviewed a while back and contributed httptest tests to?

**Comment by maelle** [comment]
[`4f70efb` (#15)](https://github.com/ropensci/rnassqs/pull/15/commits/4f70efb7f793e772297f4a50c9acd6f6f791193c)

[`3cae867` (#15)](https://github.com/ropensci/rnassqs/pull/15/commits/3cae867bba7ff7becb118609d8b0c0b23f0ab5f6)
